tests/interfaces/test_batch_jax.py

```diff
@@ -585,7 +589,7 @@ def cost_fn(a, p, device):
                 qml.expval(qml.PauliX(0))
 
             tape = tape.expand(stop_at=lambda obj: device.supports_operation(obj.name))
-            return execute([tape], device, interface="jax", **execute_kwargs)[0][0]
+            return execute([tape], device, interface=interface, **execute_kwargs)[0][0]

```

Comment:
it is super nice that you were able to add support for so many tests just by parametrizing existing ones!